cypress/e2e/swap/errors.test.ts

```diff
@@ -74,41 +90,32 @@ describe('Swap errors', () => {
       cy.get(getTestSelector('open-settings-dialog-button')).click()
       cy.get(getTestSelector('max-slippage-settings')).click()
       cy.get(getTestSelector('slippage-input')).clear().type('0.01')
-
-      // Click outside of modal to dismiss it.
-      cy.get('body').click('topRight')
+      cy.get('body').click('topRight') // close modal
       cy.get(getTestSelector('slippage-input')).should('not.exist')
 
-      // Swap 2 times.
-      const AMOUNT_TO_SWAP = 200
-      cy.get('#swap-currency-input .token-amount-input')
-        .clear()
-        .type(AMOUNT_TO_SWAP.toString())
-        .should('have.value', AMOUNT_TO_SWAP.toString())
-      cy.get('#swap-currency-output .token-amount-input').should('not.have.value', '')
-      cy.get('#swap-button').click()
-      cy.get('#confirm-swap-or-send').click()
-      cy.get(getTestSelector('dismiss-tx-confirmation')).click()
-
-      cy.get('#swap-currency-input .token-amount-input')
-        .clear()
-        .type(AMOUNT_TO_SWAP.toString())
-        .should('have.value', AMOUNT_TO_SWAP.toString())
-      cy.get('#swap-currency-output .token-amount-input').should('not.have.value', '')
-      cy.get('#swap-button').click()
-      cy.get('#confirm-swap-or-send').click()
-      cy.get(getTestSelector('dismiss-tx-confirmation')).click()
-
-      // The pending transaction indicator should reflect the state.
+      // Submit 2 transactions
+      for (let i = 0; i < 2; i++) {
```

Comment:
😱 you're breaking your own rules mate!

imo this is an exception where it's prob ok

Reply:
rules are in flux, but will be documented in a README: #6523.